packages/pg-v5/test/lib/psql.js

```diff
@@ -59,18 +59,9 @@ describe('psql', () => {
     })
 
     it('runs psql', () => {
-      let cp = sandbox.mock(require('child_process'))
```

Comment:
ensuring the we're passing the right env does seem valuable, and somebody may try to write a similar test in the future.

If we aren't worried about leaking postgres creds, maybe we could assert on a limited subset of env instead of abandoning it entirely?

```javascript
// setup
const EXPECTED_ENV = Object.freeze({
        PGAPPNAME: 'psql non-interactive',	      cp.returns(
        PGSSLMODE: 'prefer',	
        PGUSER: 'jeff',	
        PGPASSWORD: 'pass',	
        PGDATABASE: 'mydb',	
        PGPORT: 5432,	
        PGHOST: 'localhost'
});
cp.stub('spawn').callsFake((commandName, args, opts) => {
  expect(commandName).to.equal('psql');
  expect(args).to.deep.equal(['-c', 'SELECT NOW();', '--set', 'sslmode=require']);
  Object.entries(expectedEnv).forEach(([key, expectedValue]) => {
    expect(opts.env[key]).to.equal(expectedValue);
  });
  expect(opts.encoding).to.equal('utf8');
  expect(opts.stdio).to.deepEqual(['ignore, 'pipe', 'inherit']);
  return {
    /* redacted for brevity but everything from `.returns()` goes here */
    stdout: {
    }
  }
}).once();
```

then we would keep the `cp.verify()` as before

Reply:
I gave this a shot and the postgres creds are still hidden (I don't think they matter but still).
`verify()` doesn't work since that's a method on the `mock` but I was still able to safely keep the env assertions.

Thanks @fivetanley !